frontend/src/types.ts

```diff
@@ -377,7 +377,33 @@ export type Transaction = {
   fees_paid: number;
   created_at: string;
   settled_at: string | undefined;
-  metadata: unknown;
+  metadata: {
+    tlv_records: TLVRecord[];
```

Comment:
We have control over our own API - we could actually decode this on the backend

Reply:
+1 we should store the data we need it now as we have the payments table. 
also for example have a special field for the name, message metadata.

a well defined API is most important to be able to easily and flexible build features.

though I am not sure how much podcasting knowledge should be in the hub - but guess it's important.

Reply:
Done, now the API returns decoded tlv values. We already store them in metadata column, so I changed the api struct to give decoded tlv records in metadata